test/integration/create-cluster-on-hub.bats

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function token_cleanup() {
+	curl -X DELETE https://discovery-stage.hub.docker.com/v1/clusters/$1
```

Comment:
@aluzzardi Well, we can change it if we introduce something like a `swarm destroy` command in the future, so this is not an issue. Let's merge it if we need it now :+1: